Homework/Task3_1/Queue.cs

```diff
@@ -0,0 +1,73 @@
+﻿public interface IQueue<T> where T : struct
```

Comment:
Move to a separate file, one class (interface, enum, etc)-one file

Reply:
Moved to IQueue.cs